README.md

```diff
@@ -156,6 +156,7 @@ println(fibonacci(8));
 ### Syntax Highlighting
 
 * [vim-Batsh](https://github.com/vuryleo/vim-Batsh)
+* [batsh-mode](https://github.com/thechampagne/batsh-mode)
```

Comment:
I think, it'd made sense, if it were clear from the beginning, that this is an Emacs plugin. 🙂